src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+
+    public Square(double side) {
+        this.side = side;
+        setColor(new ColorSupplier().getRandomColor());
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("Figure: square, area: " + obtainTheArea() + " sq. units, side: " + side
+                + " units, color: " + getColor());
+    }
+
+    @Override
+    public double obtainTheArea() {
```

Comment:
The method name `obtainTheArea` is not following Java naming conventions. Consider renaming it to `getArea` to maintain consistency and readability.